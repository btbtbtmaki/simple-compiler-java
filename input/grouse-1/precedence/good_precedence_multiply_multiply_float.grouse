main {

  print 1.4 * 2.4 * 6.3 nl;
  print 3.0 * 4.2 * 3.8 nl;
  print 9.6 * 7.4 * 4.5 nl;
  print 5.6 * 4.6 * 8.1 nl;
  print 2.8 * 1.2 * 9.9 nl;
  print 6.6 * 9.5 * 7.4 nl;
  print 1.4 * 3.4 * 8.6 nl;
  print 4.4 * 9.2 * 9.7 nl;
  print 2.1 * 5.4 * 5.3 nl;
  print 5.0 * 7.2 * 5.7 nl;
  print 4.5 * 9.5 * 5.8 nl;
  print 1.8 * 5.3 * 4.9 nl;

  print 7.1 * 4.3 * 6.2 nl;
  print 1.1 * 2.4 * 2.6 nl;
  print 7.5 * 8.2 * 1.1 nl;
  print 5.4 * 7.1 * 6.2 nl;
  print 6.3 * 4.1 * 3.3 nl;
  print 9.4 * 9.4 * 5.8 nl;
  print 7.4 * 6.6 * 8.7 nl;
  print 4.0 * 3.4 * 1.0 nl;
  print 1.4 * 1.9 * 8.4 nl;
  print 2.7 * 5.3 * 4.8 nl;
  print 8.8 * 4.5 * 8.5 nl;
  print 6.2 * 8.6 * 2.6 nl;

  print 2.4 * 8.3 * 9.7 * 2.6 nl;
  print 3.9 * 6.6 * 8.6 * 1.8 nl;
  print 3.8 * 2.3 * 8.3 * 3.4 nl;
  print 5.2 * 6.8 * 9.1 * 2.5 nl;
  print 9.7 * 2.8 * 4.0 * 3.2 nl;
  print 5.6 * 2.7 * 8.7 * 2.5 nl;
  print 1.0 * 7.7 * 6.7 * 6.4 nl;
  print 5.8 * 8.5 * 2.3 * 1.6 nl;
  print 7.4 * 6.9 * 5.6 * 9.5 nl;
  print 3.6 * 8.3 * 8.9 * 1.3 nl;
  print 2.4 * 9.9 * 6.8 * 7.3 nl;
  print 4.5 * 5.3 * 6.4 * 3.3 nl;

  print 1.8 * 1.9 * 5.8 * 1.3 nl;
  print 1.7 * 8.5 * 4.5 * 9.8 nl;
  print 6.0 * 7.2 * 1.5 * 2.7 nl;
  print 9.1 * 9.7 * 2.6 * 4.0 nl;
  print 3.5 * 1.0 * 1.0 * 5.2 nl;
  print 7.8 * 5.5 * 5.8 * 5.2 nl;
  print 4.0 * 5.5 * 5.7 * 2.0 nl;
  print 2.7 * 6.8 * 5.5 * 3.1 nl;
  print 7.1 * 1.0 * 4.3 * 8.0 nl;
  print 6.7 * 6.9 * 5.1 * 7.0 nl;
  print 7.7 * 2.6 * 8.7 * 3.0 nl;
  print 1.7 * 1.9 * 3.9 * 9.5 nl;

}

