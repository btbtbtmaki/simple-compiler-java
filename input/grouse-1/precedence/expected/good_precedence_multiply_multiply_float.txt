21.168
47.88
319.68
208.656
33.264
463.98
40.936
392.656
60.102
205.2
247.95
46.746
189.286
6.864
67.65
237.708
85.239
512.488
424.908
13.6
22.344
68.688
336.6
138.632
502.382
398.455
246.643
804.44
347.648
328.86
330.176
181.424
2716.39
345.712
1179.45
503.712
25.7868
637.245
174.96
918.008
18.2
1293.86
250.8
313.038
244.24
1650.41
522.522
119.671
